Style StackStorm invite form

dzimine opened the pull request:
Here is a form to replace the current TypeForm one.

Suggestions are preferred in form of CSS/HTML to help my struggle with them :)

TODO:
- [x] Base layout/styles in place
- [x] Re-add and style "OK" and "Error" messages

![image](https://user-images.githubusercontent.com/1294734/28553978-81fa1488-70ab-11e7-818d-821bddd38679.png)

![image](https://user-images.githubusercontent.com/1294734/28591155-8838e8c2-7139-11e7-8eb1-3e9d2bdc9cf7.png)

![image](https://user-images.githubusercontent.com/1294734/28591095-543dd226-7139-11e7-9225-8688c07c3774.png)

LindsayHill commented on web/index.html:
```diff
@@ -13,27 +13,27 @@
+        </h1>
+        <h2 class="f5 tc f3-l fw1 mb4 near-white lh-title">Welcome to StackStorm! </h2>
+        <p class="f5 tc f3-l fw1 near-white lh-title">
+        Please introduce yourself, and leave an email <br /> to receive an invite to our Slack community.
```
Suggested text:

> Enter your email address below, to receive an invite to our Slack community. Sign up and introduce yourself!

dzimine replied:
I meant "introduce yourself" right here on the form, as a way to say "enter your name". The "name" field is also desirable. I can make it optional as @alexmakarski suggested, but somehow still care for a name.

Kami commented:
The layout is actually not all that bad at all :+1: :)

dzimine commented:
Completed, deployed, test it here https://6ktfm1psz9.execute-api.us-west-2.amazonaws.com/dev/index.html

I'll wire it to stackstorm.com/community-signup later today.

LindsayHill commented:
Couple of things: 

- Page title is "Serverless Slack Invite" - I think it should be "StackStorm Slack Invite"
- Page displayed on form submit does not replace existing page when using Safari on macOS. Instead I need to scroll to the right to see the new content. This is not obvious, especially since scroll bars are deliberately 'hidden' on macOS.

dzimine commented:
Fixed: 
- [x] Safari Lightbox issue
- [x] Missed last name (client, backend, and added tests)
- [x] Page title
Even managed to wrestle a favicon.ico there (getting it over aws api gateway is fun).

dzimine commented:
To debug (links for StackStorm folks, obviously): 

1) step-function logs here: https://us-west-2.console.aws.amazon.com/states/home?region=us-west-2#/
2) Lambda dashboard: https://us-west-2.console.aws.amazon.com/lambda/home?region=us-west-2#/ 
3) Lambda function logs (per invocation, sort by most recent): https://us-west-2.console.aws.amazon.com/cloudwatch/home?region=us-west-2#logs:prefix=/aws/lambda/slack-signup
